vmdb/app/models/container_node.rb

```diff
@@ -8,6 +8,9 @@ class ContainerNode < ActiveRecord::Base
   has_many   :container_node_conditions, :dependent => :destroy
   has_one    :computer_system, :as => :managed_entity, :dependent => :destroy
   belongs_to :lives_on, :polymorphic => true
+  has_many :containers, :through => :container_groups
+  has_many :container_services, :through => :container_groups
+  has_many :container_replicators, :through => :container_groups
```

Comment:
@simon3z @zakiva , I'm wondering, what is the use case of adding relationship between service and node? services are not *actually* running on nodes..